packages/mobile/src/utils/utils.ts

```diff
@@ -221,3 +221,9 @@ export function extractVideoId(url: string): string | null | undefined {
 export function isYoutubeURL(url: string): boolean {
   return url.includes('youtube.com') || url.includes('youtu.be');
 }
+
+export function isEmojiOnly(text: string | undefined): boolean {
+  return ((text?.match(/^\p{Extended_Pictographic}/u) || text?.match(/^\p{RI}{2}/u)) &&
```

Comment:
I'm not arguing that this would work, but perhaps we can make it match what we already have in web? That way it works/breaks on the same characters:
https://github.com/homebase-id/odin-js/blob/main/packages/chat-app/src/components/Chat/Detail/ChatMessageItem.tsx#L121